tests/app/questionnaire/test_router.py

```diff
@@ -1781,6 +1781,30 @@ def test_last_block_not_on_path(self):
         assert completed_block_not_on_path == last_completed_block_in_progress_store
         assert expected_location_url == last_location_url
 
+    @pytest.mark.usefixtures("app")
+    def test_list_collector_list_summary_returns_to_summary(self):
```

Comment:
I'm thinking instead of `list_summary_returns_to_summary` that `final_summary_returns_to_section_summary` might give that extra bit of clarity to what the test is checking - open to other thoughts or ideas